### PR TITLE
Disclaimer

### PR DESCRIPTION
**Pull request by mirhamasala**
- Create `Heading` component to be able to pass `h` tags from `Country` and `About` pages
- Use the `Disclaimer` component on both pages
- Change copy regarding `About Our Data`
- Minor text improvements
- Adapt text READ.ME accordingly

### CODE REVIEW

File: src/components/Disclaimer.vue [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
 </template>
 
 <script>
+import Heading from '@/components/Heading.vue';
+
 export default {
   name: 'Disclaimer',
+  data() {
+    return {
+      title: 'About Our Data',
```
Comment (yurikoval) [inline src/components/Disclaimer.vue:38]: Does this need to be in data?

File: src/components/Disclaimer.vue [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
-      <h3>Disclaimer</h3>
-      <p class="mb-0">
-        Advisories may change without warning. Please check official government
-        websites for up-to-date information on travel restrictions prior to your
-        journeys. This project displays information that is related only to COVID-19
-        travel restrictions and not to existing restrictions.
-      </p>
-    </div>
+  <div>
+    <Heading :tag="this.tag" :title="this.title" />
```
Comment (yurikoval) [inline src/components/Disclaimer.vue:3]: Remove `this`

File: src/views/About.vue [hunk 3]
```diff
@@ -15,24 +15,18 @@
...
         region to <a href="mailto:travel-map@tourhero.com">travel-map@tourhero.com</a>.
       </p>
 
-      <h2>Disclaimer</h2>
-      <p>Advisories may change without warning. Please check official government
-        websites for up-to-date information on travel restrictions prior to your
-        journeys. This project displays information that is related only to COVID-19
-        travel restrictions and not to existing restrictions.
-      </p>
+      <Disclaimer :tag="'h2'" />
```
Comment (yurikoval) [inline src/views/About.vue:29]: No need for binding, just pass in a string.

File: src/components/Disclaimer.vue (hunk 1 above)
Comment (mirhamasala) [reply src/components/Disclaimer.vue:38]: I can add it directly: `<Heading :tag="tag" title="About Our Data" />` ?

Comment (mirhamasala) [reply src/components/Disclaimer.vue]: Perhaps better if I add it like into `Country` and `About `together with the `tag` <Disclaimer tag="h2" title="About Our Data" />`

**Pull request merged**